dotnet/src/SemanticKernel.Abstractions/Contents/TextContent.cs

```diff
@@ -57,4 +57,13 @@ public override string ToString()
     {
         return this.Text ?? string.Empty;
     }
+
+    /// <summary>
+    /// When converting a string to a <see cref="TextContent"/>, the content is automatically set to the string value.
+    /// </summary>
+    /// <param name="text">Text content</param>
+    public static implicit operator TextContent(string text)
```

Comment:
I would probably introduce extension method that accepts `string` instead of implicit conversion, because when I invoke a method I'm able to see in IDE which method overloads exist, while I think that implicit conversion is less discoverable.

Reply:
@stephentoub Any thoughts on using implicit operators?